src/app/my-projects/my-projects.component.spec.ts

```diff
@@ -12,11 +12,20 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { MyProjectsEditComponent } from './my-projects-edit.component';
 import { MyProjectsNewComponent } from './my-projects-new.component';
 import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
-import { ReactiveFormsModule } from '@angular/forms';
+import { FormGroup, ReactiveFormsModule } from '@angular/forms';
 import { MatMomentDateModule } from '@angular/material-moment-adapter';
 import { ProjectsService } from '../projects/projects.service';
 import { Project } from '../projects/project';
 import { DeleteConfirmationDialogComponent } from '../shared/delete-confirmation-dialog/delete-confirmation-dialog.component';
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-skill-select-input',
+  template: ''
+})
+class SkillSelectInputStubComponent {
```

Comment:
Do we really need `SkillSelectInputStubComponent ` here? Is it used somehow in `MyProjectsComponent` ?